src/makeCalculator.js

```diff
@@ -37,7 +37,32 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    add(a, b) {
+      return a + b;
+    },
+    subtract(a, b) {
+      return a - b;
+    },
+    multiply(a, b) {
+      return a * b;
+    },
+    divide(a, b) {
```

Comment:
You should take only one argument here, a `number` and instead of returning it, set the appropriate value to the `result` property